packages/design-system/src/organisms/formStepHeader/index.tsx

```diff
@@ -44,15 +52,22 @@ interface StepProps {
   step: number
   title: string
   checked: boolean
+  id: string
+  setWidth: (id: string, width: number) => void
```

Comment:
Passing an imperative `setWidth` prop isn't a React way TBH. What if we allow `Step` to accept `onLayout` prop & then the consumer will decide what to do with the `LayoutChangeEvent`? I think it's a better structure for extensions